src/scheduler/task.rs

```diff
@@ -164,6 +164,13 @@ impl TaskHandlePriorityQueue {
 		self.prio_bitmap == 0
 	}
 
+	/// Checks if the given task is in the queue. Returns `true` if the task
+	/// was found.
+	pub fn find(&self, task: TaskHandle) -> bool {
```

Comment:
`contains` might be a name more consistent with `std`. What do you think?

Reply:
That sounds more fitting!